e2e/src/test/java/org.dhatim.fastexcel.benchmarks/ReaderBenchmark.java

```diff
@@ -121,22 +130,20 @@ public int streamingApachePoi() throws IOException, OpenXML4JException, SAXExcep
                         SheetContentHandler sheetHandler = new SheetContentHandler();
                         processSheet(styles, strings, sheetHandler, sheetStream);
                         assertEquals(RESULT, sheetHandler.result);
+                        return sheetHandler.result;
```

Comment:
Do we process only one sheet?

Reply:
```
    @Benchmark
    public long fastExcelReader() throws IOException {
        try (InputStream is = openResource(FILE); ReadableWorkbook wb = new ReadableWorkbook(is)) {
            Sheet sheet = wb.getFirstSheet();
```

Reply:
seems fair that if one benchmark gets just the first sheet - that other benchmarks also only get first sheet

Reply:
Indeed!